tests/sandbox/create_message_batches/test_success.py

```diff
@@ -53,7 +69,15 @@ def test_201_message_batch_valid_content_type_headers(nhsd_apim_proxy_url, conte
 @pytest.mark.parametrize('routing_plan_id', VALID_ROUTING_PLAN_ID)
 def test_201_message_batch_valid_routing_plan_id(nhsd_apim_proxy_url, routing_plan_id):
     """
-    .. py:function:: Test valid routing plans
+    .. py:function:: Scenario: A user submitting a request with a \
+        valid routing plan header receives a 201 response
+
+        | **Given** the user provides a valid routing plan
```

Comment:
**Given** the user provides a valid routing plan when creating a batch of messages